data/html/de/index.html

```diff
@@ -13,7 +13,7 @@ <h1><img src="../images/oi-logo.png" width=300 height=80
 
 <h2>Willkommen bei OpenIndiana Hipster!</h2>
 
-<p>OpenIndiana ist eine auf Illumos basierende Unix-ähnliche Distribution, die von OpenSolaris abgeleitet wurde.
+<p>OpenIndiana ist eine auf illumos basierende Unix-ähnliche Distribution, die von OpenSolaris abgeleitet wurde.
```

Comment:
There is a comma missing because it's an equally ranked enumation that needs to be comma separated in German:
"ist eine auf illumos basierende, Unix-ähnliche Distribution,..."

Reply:
Added the comma for the enumeration

Reply:
Hi, I have not used Github before, so i don't know what I have to do now :/

Reply:
If nobody else has objections (it seems so), you should squash the commits into a single one.
Then the PR can be merged.